src/backend/distributed/shared_library_init.c

```diff
@@ -1069,6 +1070,17 @@ RegisterCitusConfigVariables(void)
 		GUC_NO_SHOW_ALL,
 		NULL, NULL, NULL);
 
+	DefineCustomBoolVariable(
+		"citus.enable_warnings_propagation",
```

Comment:
This name is too generic.

maybe `citus.enable_unsupported_feature_messages` or such?